Enable RHOAI components in edu cluster

IsaiahStapleton opened the pull request:
This adds the same DataScienceCluster spec that prod cluster uses which enables different RHOAI components. One of which, Kueue, is necessary for running Jonathan's GPU course.

DanNiESh commented on cluster-scope/overlays/nerc-ocp-edu/rhoai/datasciencecluster.yaml:
```diff
@@ -21,15 +17,22 @@ spec:
+    ray:
+      managementState: Managed
+    kueue:
       managementState: Removed
```
Should it be `Managed`?

IsaiahStapleton replied:
Not sure how I missed that lol. Yes it should be.

IsaiahStapleton replied:
Didn't mean to close the issue, but this has been fixed Kueue is set to managed.

IsaiahStapleton committed:
Enable RHOAI components in edu cluster

This adds the same DataScienceCluster spec that prod cluster uses which enables different RHOAI components. One of which, Kueue, is necessary for running Jonathan's GPU course.

Signed-off-by: Isaiah Stapleton <istaplet@redhat.com>
